src/styles/clock.scss

```diff
@@ -0,0 +1,42 @@
+.clock {
+  position: relative;
+  left: 50%;
+  top: 50%;
+
+  @include clockSize(500px);
+
+  border-radius: 50%;
+  background: #fff url(../images/ios_clock.svg) no-repeat center;
+  background-size: 88%;
+
+  transform: translate(-50%, 50%);
+
+  &:after {
+    background-color: #000;
+    border-radius: 50%;
+    content: "";
```

Comment:
for what it?

Reply:
This is a dot in the center of my watch)

Reply:
nice)